### PR TITLE
add item number in multiple choice item

### LINKED ISSUE #135: Problem Number In Multiple Choice Item
We need problem number information in multiple choice item.

### PR DESCRIPTION
**Pull request by jauhararifin**
Resolve #135 

[#135](https://github.com/ia-toki/judgels/issues/135) Add item number in `Bundle/ProblemStatementCard`

### CODE REVIEW

**Review by fushar — APPROVED** [review]

File: judgels-frontends/raphael/src/components/ProblemWorksheetCard/Bundle/ProblemStatementCard/ProblemStatementCard.tsx [hunk 1]
```diff
@@ -58,7 +60,8 @@ export class ProblemStatementCard extends React.Component<ProblemStatementCardPr
           if (item.type === ItemType.Statement) {
             return this.renderStatement(item);
           } else {
-            return this.renderMultipleChoice(item);
+            multipleChoiceItemNum++;
+            return this.renderMultipleChoice(item, multipleChoiceItemNum);
```
Comment (fushar) [inline judgels-frontends/raphael/src/components/ProblemWorksheetCard/Bundle/ProblemStatementCard/ProblemStatementCard.tsx]: Wait, can this just use the new `item.number`? @nathanchrs ?

**Review by fushar — CHANGES_REQUESTED** [review]

File: judgels-frontends/raphael/src/components/ProblemWorksheetCard/Bundle/ProblemStatementCard/ProblemStatementCard.tsx (hunk 1 above)
Comment (nathanchrs) [reply judgels-frontends/raphael/src/components/ProblemWorksheetCard/Bundle/ProblemStatementCard/ProblemStatementCard.tsx]: Yes, `Item` have a `number` field now.

Comment (fushar) [reply judgels-frontends/raphael/src/components/ProblemWorksheetCard/Bundle/ProblemStatementCard/ProblemStatementCard.tsx]: @jauhararifin Please update

Comment (jauhararifin) [reply judgels-frontends/raphael/src/components/ProblemWorksheetCard/Bundle/ProblemStatementCard/ProblemStatementCard.tsx]: Updated

**Comment by jauhararifin** [comment]
This PR also add item number in submissions page.

**Review by fushar — APPROVED** [review]

**Pull request merged**